lms/static/js/groups/views/cohort_form.js

```diff
@@ -0,0 +1,138 @@
+var edx = edx || {};
+
+(function($, _, Backbone, gettext, interpolate_text, CohortModel, NotificationModel, NotificationView) {
+    'use strict';
+
+    edx.groups = edx.groups || {};
+
+    edx.groups.CohortFormView = Backbone.View.extend({
+        events : {
+            'change .cohort-management-details-association-course input': 'onRadioButtonChange',
+            'click .tab-content-settings .action-save': 'saveSettings',
+            'submit .cohort-management-group-add-form': 'addStudents'
+        },
+
+        initialize: function(options) {
+            this.template = _.template($('#cohort-form-tpl').text());
+            this.contentGroups = options.contentGroups;
+            this.context = options.context;
+        },
+
+        showNotification: function(options, beforeElement) {
+            var model = new NotificationModel(options);
+            this.removeNotification();
+            this.notification = new NotificationView({
+                model: model
+            });
+            this.notification.render();
+            beforeElement.before(this.notification.$el);
+        },
+
+        removeNotification: function() {
+            if (this.notification) {
+                this.notification.remove();
+            }
+        },
+
+        render: function() {
+            this.$el.html(this.template({
+                cohort: this.model,
+                contentGroups: this.contentGroups,
+                studioGroupConfigurationsUrl: this.context.studioGroupConfigurationsUrl
+            }));
+            return this;
+        },
+
+        onRadioButtonChange: function(event) {
+            var target = $(event.currentTarget),
+                groupsEnabled = target.val() === 'yes';
+            if (!groupsEnabled) {
+                // If the user has chosen 'no', then clear the selection by setting
+                // it to the first option ('Choose a content group to associate').
```

Comment:
Change the comment to reference the correct text (or something generic enough so that it doesn't depend on the text).